upgrade_tests/helpers/scenarios_utils.py

```diff
@@ -0,0 +1,181 @@
+"""All the helper functions, needed for scenarios test case automation to be
+added here"""
+import os
+import pickle
+import sys
+import time
+
+from automation_tools import manage_daemon
+from automation_tools.bz import bz_bug_is_open
+from automation_tools.satellite6.hammer import (
+    get_attribute_value,
+    hammer,
+)
+from upgrade.helpers.docker import remove_all_docker_containers
+from upgrade.helpers.rhevm import (
+    create_rhevm_instance,
+    get_rhevm_client,
+    wait_till_rhevm_instance_status
+)
+from automation_tools.satellite6.upgrade.tools import logger
+from fabric.api import execute, run
+
+rpm1 = 'https://inecas.fedorapeople.org/fakerepos/zoo3/bear-4.1-1.noarch.rpm'
+rpm2 = 'https://inecas.fedorapeople.org/fakerepos/zoo3/camel-0.1-1.noarch.rpm'
+data = {}
+logger = logger()
+
+
+def create_dict(entities_dict):
+    """Stores a dictionary of entities created in satellite on the disk
+
+        :param string entities_dict: A dictionary of entities created in
+        satellite
+        """
+    data.update(entities_dict)
+    with open('entities_data', 'wb') as pref:
```

Comment:
We need a separate PR for this to collect all tests entities data in one file. This will overwrite the last test data. And we should append the entity of test case and not write again.